src/api/spec/support/database_cleaner.rb

```diff
@@ -1,6 +1,7 @@
 require 'database_cleaner'
 
 RSpec.configure do |config|
+  # To prevent uncommitted transactions being used in JavaScript-dependent specs
```

Comment:
This is not really the reason we are not using tranactional fixtures. We are not using them because we are using `factory_girl` to set up everything the test needs up front, instead of loading a set of fixtures in the beginning of the suite.